src/Fixer/Import/OrderedImportsFixer.php

```diff
@@ -68,10 +68,12 @@
     /**
      * Configuration settings for sort algorithm and sor types.
      *
+     * Use 'null' for sorting alphabetically by default.
```

Comment:
But I am setting default here: https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/2280/files#diff-80d6131ebed1ab07edef75d2d7ad6c82R59 

Or you think it's better to do like this:

```php
if (null === $configuration) {
    self::$sortAlgorithm = self::SORT_ALPHA;
    return;
}
```

Or remove the comment ?